doc/user/spark.md

```diff
@@ -21,7 +21,9 @@ $ cd daos
 $ git checkout <desired branch or commit>
 ## assume DAOS is built and installed to <daos_install> directory
 $ cd src/client/java
-$ mvn clean package -DskipITs -Ddaos.install.path=<daos_install>
+## with-proto3-netty4-deps profile builds jars with protobuf 3 and netty-buffer 4 shaded
+## It spares you potential third-party jar compatibility issue.
+$ mvn -Pdistribute,with-proto3-netty4-deps clean package -DskipTests -Ddaos.install.path=<daos_install>
```

Comment:
Is the change from "-DskipITs" to "-DskipTests" intended?

Reply:
yes. since all tests have been run in PR already. there is no need to re-run at delivery time.